tests/test_crs.py

```diff
@@ -291,9 +288,7 @@ def test_epsg__no_code_available():
 def test_crs_OSR_equivalence():
     crs1 = CRS.from_string('+proj=longlat +datum=WGS84 +no_defs')
     crs2 = CRS.from_string('+proj=latlong +datum=WGS84 +no_defs')
-    crs3 = CRS({'init': 'epsg:4326'})
     assert crs1 == crs2
```

Comment:
🤔 I'm not sure to understand how this test can success if we ignore axis order (now default to `NO`)

Reply:
🤔 ok so both CRS will return `CRS.from_epsg(4326)` which is why they are `==`

@sgillies So going back to my other comment, can we add a test for `CRS.from_epsg(4326) == CRS.from_string("OGC:CRS84")`